dedis-cothority.toml

```diff
@@ -1,99 +1,97 @@
 [[servers]]
   Address = "tls://conode.dedis.ch:7000"
-  Url = "https://conode.dedis.ch"
   Suite = "Ed25519"
   Public = "ec5c65a3c922d1df32075640e3de606197be24af76059a2ef145501122884bd3"
   Description = "EPFL Cothority-server"
+  URL = "https://conode.dedis.ch"
```

Comment:
Is this change of case ("Url" → "URL") intentional? It seems to break `Roster.fromTOML()`...

Reply:
Oups, I overlooked it; it seems that the golang serialisation uses [`URL`](https://github.com/dedis/onet/blob/master/app/config.go#L211) and that the typescript one uses [`Url`](https://github.com/dedis/cothority/blob/master/external/js/cothority/src/network/proto.ts#L59).

Adding a commit fixing it in external/js.

Reply:
Yes, as far as I know, the specs in my head say that both should be accepted. It's unfortunate, but, well, that's what it is...

Reply:
Looks like https://github.com/BurntSushi/toml allow anycase for the keys, even `uRl`...
So, updating the external/js patch, to allow for both casing, with preference to `URL`.

Reply:
The [TOML spec](https://github.com/toml-lang/toml#spec) seems to be case-sensitive though.

Reply:
Correct.. Maybe we should drop this dep at some point, as upstream is dead since nearly 2 year, and it wrongly implements an old version of TOML.